### PR TITLE
added logo component to appbar

### PR DESCRIPTION
**Pull request by ilkkamaksy**
Added logo component with logo image changing with the theme

### CODE REVIEW

**Comment by jakekall** [comment]
First of all, the logo looks great!
Secondly, the Main menu link should probably be removed since clicking on the logo does the same thing.
Thirdly, can you change the background of the dark logo to rgb(33,33,33) so that it blends perfectly with the header?

**Comment by ilkkamaksy** [comment]
Switched to transparent png files and removed main menu link.

**Review by jakekall — APPROVED** [review]

File: frontend/src/components/Logo.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    createStyles,
+    makeStyles,
+    Link,
+  } from '@material-ui/core'
+
+interface Props {
+  theme: string
+}
+
+const stylesInUse = makeStyles((theme) =>
```
Comment (guotin) [inline frontend/src/components/Logo.tsx:13]: Parameter never used or am I missing something?

Comment (guotin) [reply frontend/src/components/Logo.tsx:13]: I can't see any difference in the application when it's removed

Comment (jakekall) [reply frontend/src/components/Logo.tsx:13]: The `theme` parameter is only needed if we have to directly access the theme given to `<ThemeProvider>` in App. For example in RegisterForm it's used to get the colors for error and success messages. The theme parameter is also unnecessary in Header and BranchSelector components and can be removed.

Comment (ilkkamaksy) [reply frontend/src/components/Logo.tsx]: parameter now removed

**Review by guotin — APPROVED** [review]

**Pull request merged**